internal/providers/providers.go

```diff
@@ -55,14 +56,14 @@ func GetProviderBuilder(
 		return nil, fmt.Errorf("error decrypting access token: %w", err)
 	}
 
-	return NewProviderBuilder(&prov, encToken, decryptedToken.AccessToken, opts...), nil
+	return NewProviderBuilder(&prov, encToken.OwnerFilter, decryptedToken.AccessToken, opts...), nil
 }
 
 // ProviderBuilder is a utility struct which allows for the creation of
 // provider clients.
 type ProviderBuilder struct {
 	p               *db.Provider
-	tokenInf        db.ProviderAccessToken
+	ownerFilter     sql.NullString // NOTE: we don't seem to actually use the null-ness anywhere.
```

Comment:
we'll get rid of ownerFilter at some point. This is more of a provider specific configuration rather than for the token, but it ended up there because the token table came first.

Reply:
I suspect we want to remove SQL-isms from the provider interface at some point, but I didn't want to boil that ocean here.  At the same time, the `ownerFilter` was the only part of the access token which was being used, so I wanted to narrow that interface to exclude the other rows.

Reply:
that's fine, it is what it is.